### ISSUE TITLE
Question : do you freeze the darknet53 backbone when training?

### ISSUE THREAD

**Issue by pfeatherstone** [AlexeyAB/darknet#5185]
Do you freeze the darknet53 backbone when training? 
Or do you load the backbone with the pretrained weights then allow it to train as normal?
Or do you load the backbone with the pretrained weights and give it a lower learning rate than the rest of the network, then all it to train?

**Comment by AlexeyAB** [comment]
> Or do you load the backbone with the pretrained weights then allow it to train as normal?

Yes.

**Comment by pfeatherstone** [comment]
thanks
